src/shared/constants/common.ts

```diff
@@ -283,6 +283,10 @@ export const ADMIN_ERROR_MAP = {
     Code: 'E003',
     Message: 'Invalid aura address format',
   },
+  INVALID_NAME_TAG: {
+    Code: 'E004',
+    Message: 'Invalid aura name tag format',
```

Comment:
chỗ này nên chi tiết hơn để user biết nhập sai gì ạ

Reply:
Để câu message này thì a thấy hơi dài
Name tag only accept normal character (Upper case, Lower Case, Number), not accept special character except dot (.), dash (-), underscore (_), space ( ).

Reply:
Có lẽ tooptip hướng dẫn sẽ tốt hơn, truyền vào message error cái hướng dẫn cách nhập đúng. Mà cái này admin sài chắc họ sẽ biết rule.